docs/topics/db/models.txt

```diff
@@ -1238,6 +1238,11 @@ number of proxy models that share a common non-abstract parent class.
     In earlier versions, a proxy model couldn't inherit more than one proxy
     model that shared the same parent class.
 
+.. note::
+
+    The ``ordering`` and ``get_latest_by`` meta attributes are still inherited
```

Comment:
Maybe it would be better to say something like "Proxy model `Meta` inheritance works the same as described in https://docs.djangoproject.com/en/dev/topics/db/models/#meta-and-multi-table-inheritance" so that if things change, we don't have to update two places.